src/openrct2/ride/gentle/meta/Circus.h

```diff
@@ -54,5 +54,8 @@ constexpr const RideTypeDescriptor CircusRTD =
     SET_FIELD(StartRideMusic, OpenRCT2::RideAudio::CircusStartRideMusicChannel),
     SET_FIELD(DesignCreateMode, TrackDesignCreateMode::Default),
     SET_FIELD(MusicUpdateFunction, CircusMusicUpdate),
+    SET_FIELD(Classification,RideClassification::Ride),
+    SET_FIELD(UpdateLeaveEntrance,PeepUpdateRideLeaveEntranceDefault),
+    SET_FIELD(MusicTrackOffsetLength, OpenRCT2::RideAudio::RideMusicGetTrackOffsetLength_Circus),
```

Comment:
```suggestion
    SET_FIELD(MusicTrackOffsetLength, OpenRCT2::RideAudio::RideMusicGetTrackOffsetLength_Circus),
```

Shouldn't this be just this single line change?

Reply:
I'm not too sure why the lines got removed but they are still there...

Reply:
The lines are still there though, I'm not sure why they show red.